upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -756,6 +756,33 @@ if [[ ${state_recorded} == "0" && $(hostname) == "${PRIMARY_NODE}" ]]; then
       fi
     fi
 
+    # cert-manager will need to be upgraded if cray-drydock version is less than 2.18.4.
+    # This will only be the case in some CSM 1.6 to CSM 1.6 upgrades.
+    # It only needs to be checked if cert-manager is not already being upgraded.
+    if [ "${needs_upgrade}" -eq 0 ]; then
+      drydock_vers=$(helm get metadata -n loftsman cray-drydock -o json | jq -r '.version')
```

Comment:
This is a bit ugly, but it should work with older versions:
```
drydock_vers=$(helm get values -n loftsman cray-drydock | grep version: | sed 's/ *version: //')
```

Reply:
I tested that command on helm-3.11 and helm-3.13 and it worked on both. I updated prerequisites.sh with it. Thanks!